src/SplitText.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export const SplitText = ({ children, className, separator, ...rest }) => {
+    const words = children.split(separator);
+
+    return words.map((word, index) => {
+        const withSeparator = index !== words.length - 1;
+
+        return (
+            <span
+                key={ `word-${index}` }
+                className={ classNames('react-split-text', className) }
+                { ...rest }>
+                { word }
+                { withSeparator && separator }
+            </span>
+        );
+    });
+};
+
+SplitText.propTypes = {
+    children: PropTypes.any.isRequired,
```

Comment:
`children` must be of type `string`, right?

Reply:
Yup, this was a copy-paste and I forgot to change 😬